packages/jbrowse-web/src/plugins/LinearGenomeView/models/index.js

```diff
@@ -102,7 +100,22 @@ export default function LinearGenomeViewStateFactory(pluginManager) {
         return getConf(self, 'reversed')
       },
     }))
+    .volatile(() => ({
+      displayedRegions: [],
```

Comment:
do you really think displayedRegions should really be volatile? we probably would want to save it so people can get back to what they were viewing

Reply:
I see this as getting moved to whatever we decide is going to keep track of the displayed regions (#197).